src/config/config.ts

```diff
@@ -118,23 +119,23 @@ const config: Config = {
 
   apiKey: '',
 
+  apiUrl: process.env.API_URL ?? 'https://wakatime.com/api',
+
   colors: {
     allGood: '',
     lightTheme: 'white',
     notLogging: 'gray',
     notSignedIn: 'red',
   },
 
-  currentUserApiUrl:
-    process.env.CURRENT_USER_API_URL ?? 'https://wakatime.com/api/v1/users/current',
+  currentUserApiEndPoint: process.env.CURRENT_USER_API_URL ?? '/v1/users/current',
 
   detectionIntervalInSeconds: 60,
 
   devSites:
     'https://codepen.io/\nhttps://www.codewars.com/\nhttps://dev.to/\nhttps://github.com/\nhttps://www.hackerrank.com/\nhttps://leetcode.com/\nhttps://developer.mozilla.org/en-US/\nhttps://stackoverflow.com/\nhttps://www.udemy.com/\nhttps://www.w3schools.com/',
 
-  heartbeatApiUrl:
-    process.env.HEART_BEAT_API_URL ?? 'https://wakatime.com/api/v1/users/current/heartbeats',
+  heartbeatApiEndPoint: process.env.HEART_BEAT_API_URL ?? '/v1/users/current/heartbeats',
```

Comment:
Heartbeat is a single word, maybe renaming it now makes sense?

Reply:
Done